invenio_rdm_records/services/schemas/record.py

```diff
@@ -88,3 +90,37 @@ def default_nested(self, data, many, **kwargs):
             data["custom_fields"] = {}
 
         return data
+
+    def hide_tombstone_or_metadata(self, data):
+        """Hide tombstone info if the record isn't deleted and metadata if it is."""
+        if not data.get("deletion_status", {}).get("is_deleted", False):
+            data.pop("tombstone", None)
+        else:
+            data.pop("access", None)
+            data.pop("created", None)
+            data.pop("custom_fields", None)
+            data.pop("files", None)
+            data.pop("is_deleted", None)
+            data.pop("is_draft", None)
+            data.pop("is_published", None)
+            data.pop("media_files", None)
+            data.pop("metadata", None)
+            data.pop("parent", None)
+            data.pop("revision_id", None)
+            data.pop("stats", None)
+            data.pop("status", None)
+            data.pop("updated", None)
+            data.pop("versions", None)
+
+            data["tombstone"]["identifiers"] = [
+                {"scheme": k, "identifier": v} for k, v in data.get("pids", {}).items()
+            ]
```

Comment:
this basically just duplicates the information already available in `record['identifiers']`, massaged into a slightly more digestible format.
should we keep this around and/or throw `record['identifiers']` out, if the record is deleted?